tests/unit/config_tests.py

```diff
@@ -46,13 +46,19 @@ def test_parse_arguments_list_WithLongArgument_ShouldParseArgumentValue(self, ar
         (['build-packages'], 'packages_metadata_repo_url',
          'https://github.com/open-power-host-os/versions.git'),
         (['build-packages'], 'packages_metadata_repo_branch', 'master'),
-        (['build-packages'], 'mock_args', ''),
+        (['build-packages'], 'mock_args', '--enable-plugin=tmpfs '
+         '--plugin-option=tmpfs:keep_mounted=True --plugin-option='
+         'tmpfs:max_fs_size=32g --plugin-option=tmpfs:required_ram_mb=39800 '
+         '--verbose'),
```

Comment:
The defaults in `config.py` were not matching the values in `host_os.yaml` for `mock_args`, so the tests would fail if not updated.